sympy/geometry/ellipse.py

```diff
@@ -1605,6 +1606,13 @@ def __new__(cls, *args, **kwargs):
 
             raise GeometryError("Circle.__new__ received unknown arguments")
 
+    def _eval_evalf(self, prec=15, **options):
+        pt, r = self.args
+        dps = prec_to_dps(prec)
+        pt = pt.evalf(n=dps, **options)
+        r = r.evalf(n=dps, **options)
+        return self.func(pt, r, evaluate=False)
```

Comment:
@smichr Can this be written something like `self.func(*[x.evalf(n=prec_to_dps(prec), **options) for x in self.args], evaluate = False)` similar to one in Point class ?

Reply:
This method computes `dps` only once which is good. Ideally `_eval_evalf` would be able to call a method that takes binary precision. There is a PR for that but it is unfinished  #20493